src/librssguard/CMakeLists.txt

```diff
@@ -623,14 +623,20 @@ if(APPLE)
   )
 elseif(WIN32)
   target_link_libraries(rssguard PUBLIC
-    Shell32.lib
+    shell32
     odbc32
   )
 endif()
 
-if(WIN32 OR OS2)
+if(MSVC OR OS2)
   install(TARGETS rssguard DESTINATION .)
   set(HEADERS_FOLDER "include/librssguard")
+elseif(MINGW)
```

Comment:
The final application file/folder layout must be the same, regardless of used compiler. I do not want any subfolders etc.